templates/taxi/driver_list.html

```diff
@@ -5,10 +5,17 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
+        {% if user.id == driver.id %}
+          <li>
+            <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
```

Comment:
You can add "(me)" part in the if statement after the link, so you don't need to repeat "a href.." twice